ansible/roles/kafka-logging-setup/files/kafka-lv-brokers.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "local.storage.openshift.io/v1"
```

Comment:
We should delete this file since it won't be in use anymore.

Reply:
deleted